src/WooCommerce.php

```diff
@@ -388,6 +389,27 @@ public static function woocommerce_product_options_general_product_data() {
     }
   }
 
+
+  /**
+   * Displays custom fields for single products on the invenotry tab.
+   *
+   * @implements woocommerce_product_options_stock_fields
+   * @implements woocommerce_product_after_variable_attributes
+   */
+  public static function showIncomingstock() {
+    if (ProductFieldsManager::show_field(self::FIELD_PRODUCT_INCOMING_STOCK)) {
+      // Back in stock date field.
+      echo '<div class="options_group show_if_simple show_if_external">';
+      woocommerce_wp_text_input([
+        'id'          => self::FIELD_PRODUCT_INCOMING_STOCK,
+        'label'       => __('Incoming Stock', Plugin::L10N),
+        'desc_tip'    => 'true',
+        'description' => self::get_product_fields()[self::FIELD_PRODUCT_INCOMING_STOCK],
```

Comment:
Seems the checkbox in `woocommerce_product_options_related` is using wrong formatting as well, but overall I think this file uses our usual conventions.

```suggestion
        'id' => self::FIELD_PRODUCT_INCOMING_STOCK,
        'label' => __('Incoming Stock', Plugin::L10N),
        'desc_tip'  => 'true',
        'description' => self::get_product_fields()[self::FIELD_PRODUCT_INCOMING_STOCK],
```